cmd/rofl/deploy.go

```diff
@@ -233,13 +236,34 @@ var (
 				acc := common.LoadAccount(cfg, npa.AccountName)
 				var sigTx, meta any
 				sigTx, meta, err = common.SignParaTimeTransaction(ctx, npa, acc, conn, tx, nil)
+				if strings.Contains(err.Error(), "instance not found") {
+					if deployReplaceMachine {
+						fmt.Printf("Machine instance not found. Obtaining new one...")
+						machine, err = obtainMachine()
+						cobra.CheckErr(err)
+						return doDeployMachine(machine)
+					}
+
+					cobra.CheckErr("Machine instance not found. If your instance expired, run this command with --replace-machine flag to replace it with a new machine.")
```

Comment:
```suggestion
					cobra.CheckErr("Machine instance not found.\nHint: If your instance expired, run this command with --replace-machine flag to replace it with a new machine.")
```

Reply:
We use "tips" when we try to guess why the error happened, see https://github.com/oasisprotocol/cli/blob/master/cmd/common/transaction.go#L573-L597

Reply:
OK, I've changed it :)  But note that if this error is ever moved outside of `cobra.CheckErr` (e.g. by returning it and/or formatting it with `fmt.Errorf`), the linter will complain about the newline and the capitalization.  We could disable that linter check if we prefer this style.